src/software/simulated_tests/ball_speed_threshold.py

```diff
@@ -0,0 +1,52 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class BallSpeedThreshold(Validation):
```

Comment:
Nit: Can we mention the units of speed somewhere?

Reply:
added comments